docs/about/faq.md

```diff
@@ -44,6 +44,15 @@ For convenience, we also [build container images with headscale](../setup/instal
 we don't officially support deploying headscale using Docker**. On our [Discord server](https://discord.gg/c84AZQhmpx)
 we have a "docker-issues" channel where you can ask for Docker-specific help to the community.
 
+## What is the recommended update path? Can I skip multiple versions while updating?
+
+Please follow the steps outlined in the [upgrade guide](../setup/upgrade.md) to update your existing Headscale
+installation. Its best to update from one stable version to the next (e.g. 0.24.0 &rarr; 0.25.1 &rarr; 0.26.1) in case
+you are multiple releases behind. You may pick the latest available patch release.
```

Comment:
Should we was «should always»?

there was one release where we had a patch release with a broken migration so it would have been the best if people took the latest.

Reply:
Changed to `You should always pick the latest available patch release.`